src/summary/profile/demographics-card.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import SummaryCard from "../cards/summary-card.component";
+import { match } from "react-router";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import dayjs from "dayjs";
+
+export default function DemographicsCard(props: DemographicsCardProps) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient =>
+      setCurrentPatient(patient)
+    );
+
+    return () => subscription.unsubscribe();
+  });
+
+  return (
+    <SummaryCard name="Demographics" match={props.match}>
+      <SummaryCardRow linkTo="/">
```

Comment:
These rows will eventually be links that go places?

Reply:
Yep, exactly. This PR doesn't implement this links yet. See https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart---Master-(current-file)?node-id=517%3A722 for how each row has its own little right caret `>` and can be clicked individually